### PR TITLE
#156100328: onboard new user to the game

### PR DESCRIPTION
**Pull request by omokolataiwo**
#### What does this PR do?
This PR onboard new user to the game thorough product tour.

#### Description of Task to be completed?
- [x] onboard new user with hopscotch
- [x] test ui with protractor
- [x] add html tag to take tour again

#### How should this be manually tested?
- Sign or Signup
- If you have not taken tour before, tour should pop up automatically
- If you have taken tour, tour should not come up, except user click on take tour again

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[finishes: #156100328]

#### Screenshots (if appropriate)
screen shot 2018-04-12 at 7 12 11 am
screen shot 2018-04-12 at 7 16 28 am

### CODE REVIEW

File: public/views/app.html [hunk 1]
```diff
@@ -6,12 +6,16 @@
       <span id="second-word"> for</span>
       <span id="third-word">Humanity</span>
     </div>
-    <div id = 'abandon-game-button' ng-click="abandonGame()">
-        Abandon Game
+    <div id='abandon-game-button' ng-click="abandonGame()">
+      Abandon Gam
```
Comment (morekoya) [inline public/views/app.html]: Please note incorrect spelling

File: test/e2e/conf.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  framework: 'jasmine',
+  specs: ['onboarding.spec.js'],
+
+  onPrepare: () => {
+    require('babel-register');
+    require('babel-core/register')({ presets: ['es2015'] });
+
+    browser.driver.get('http://localhost:3000');
+    /* Sign up
+    browser.driver.findElement(by.id('sign-up-btn-landing')).click();
```
Comment (morekoya) [inline test/e2e/conf.js]: Is this obsolete code?

Comment (omokolataiwo) [reply test/e2e/conf.js]: I guess the code is obsolete. I actually used it to register an account.

File: public/views/app.html (hunk 1 above)
Comment (omokolataiwo) [reply public/views/app.html]: Noted

**Pull request merged**